app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
> isn't the presence of a note_for_later sufficient to assume that it was once saved for later?

We're talking about two different things now, I think.  

When an application is saved for later, it need not have a `note`.  This block of code marks an activate application that was at one point `saved_for_later`, but is reliant on the fact that it included a `note`.  I wanted to be able to say "the application was saved for later" regardless of whether it ever had a `note`.  If it didn't have a `note` and currently is _not_ `saved_for_later`, there's no way (except perhaps with `paper_trial`, like you said), to know whether it was saved for later.

So, the presence of a `note_for_later` *is* enough to assume it was once `saved_for_later`, but this would cause us to miss apps that were saved but didn't have a `note.`  Is that clearer?